### DISCUSSION TITLE
`useLazyAsyncData` not fetch on ssr

### DISCUSSION THREAD

**Discussion by sotnikovse** [nuxt/nuxt#16586] [Questions]
### Environment
- Operating System: `Darwin`
- Node Version:     `v16.16.0`
- Nuxt Version:     `3.0.0-rc.4`
- Nitro Version:    `0.4.24`
- Package Manager:  `yarn@1.22.19`
- Builder:          `vite`
- User Config:      `-`
- Runtime Modules:  `-`
- Build Modules:    `-`

### Reproduction
https://stackblitz.com/edit/github-fofzrx-isja3s?file=pages%2Fdata.vue,pages%2Fdata-lazy.vue

1. in page have two `useLazyAsyncData`, `post` request and `user` request
2. `user` request depends on `post` request data
3. open `/data` route on ssr
4. `post` request fetched on server as expected
5. `user` request not fetched
<img width="1319" alt="Screenshot 2022-10-14 at 23 03 23" src="https://user-images.githubusercontent.com/16667756/195866345-f509e711-ce04-4ad8-9de0-b4a8a21dc0fb.png">

on client works as expected.

with `await useLazyAsyncData` `post` request(see `/data-lazy` route on [stackblitz](https://stackblitz.com/edit/github-fofzrx-isja3s?file=pages%2Fdata.vue,pages%2Fdata-lazy.vue)) works as expected.

is this a bug or in this case `useLazyAsyncData` should be used with await?

**Comment by hidekatsu-izuno** [comment]
the execution of useLazyAsyncData/useLazyFetch is blocked when server option is true (that is a default value).

The server option should be false automatically if the lazy option is true.

**Reply by sotnikovse** [reply]
`server: false` it's not what i need, in example see /data-client route, it's work as expected on both mode, but i want fetch second request on server.

in example without `await`(/data route) on client mode work as expected, on server mode request with `watch` not fetched, with `await`(/data-lazy route) in both mode works as expected, no blocked on client mode, block and fetch both requests on server mode.

i'm just not sure, it's ok or it's a bug and should not use with await.
